caravel/models.py

```diff
@@ -1574,9 +1577,14 @@ def wrapper(*args, **kwargs):
             d.update(kwargs)
             slice_id = d.get('slice_id', 0)
             slice_id = int(slice_id) if slice_id else 0
+            params = ""
+            try:
+                params = json.dumps(d)
```

Comment:
what about trying to apply str(arg) to every arg rather than catching the exception here?